datajunction-server/tests/api/collections_test.py

```diff
@@ -0,0 +1,208 @@
+# pylint: disable=too-many-lines
+"""
+Tests for the collections API.
+"""
+import pytest
+from httpx import AsyncClient
+
+
+class TestDataForNode:
+    """
+    Test ``POST /collections/``.
+    """
+
+    @pytest.mark.asyncio
+    async def test_creating_a_collection(
+        self,
+        client_with_account_revenue: AsyncClient,
```

Comment:
I know this may add some work to your PR, but it would be nice to use the module scope fixtures on the new code already. It is easier to modify the tests when you wrote them than for a new person to learn how they are designed. No pressure. 

If you decide to do so, the patter is simple... use the same fixture but with the `module__` prefix (it should exist already) and try to make it work with your tests.

Reply:
That's right, I forgot about this new pattern. Thanks @agorajek!